### ISSUE TITLE
Wibble Log Events

### ISSUE THREAD

**Issue by Marphies** [freescout-helpdesk/freescout#3720]
The "Wibble" Log is full with event entries like this:
`Jan 22, 20:51:18 | {"meta":null,"subject":"[#123456] Message Title"}`

Is this meant to be like that?

> PHP version: PHP 8.1.11
> FreeScout version: 1.8.118
> Database: Mysql (10.6.12-MariaDB-0ubuntu0.22.04.1)
> Are you using CloudFlare: No

**Comment by freescout-helpdesk** [comment]
What is "Wibble" log? Is it form some custom module?

**Comment by Marphies** [comment]
We use the following custom modules, none of which seem related to this. Screenshot below.

- https://github.com/LJPc-solutions/freescout-calendar-module
- https://github.com/LJPc-solutions/freescout-alternative-signatures-module
- https://github.com/aarhus/UnSub


![grafik](https://github.com/freescout-helpdesk/freescout/assets/147406642/db26698c-0a73-4eaa-ae4c-8a99b01872d9)

**Comment by Marphies** [comment]
You are right. This log is coming from the extension https://github.com/aarhus/UnSub
After disabling it and deleting the log it is gone. Thank you.
